src/buildHTML.js

```diff
@@ -547,14 +547,40 @@ groupTypes.genfrac = function(group, options) {
         options);
 };
 
+/**
+ * Parse a `sizeValue`, as parsed by functions.js argType "size", into
+ * a CSS em value. `options` gives the current options.
+ */
 const calculateSize = function(sizeValue, options) {
-    let x = sizeValue.number;
-    if (sizeValue.unit === "ex") {
-        x *= options.fontMetrics().emPerEx;
-    } else if (sizeValue.unit === "mu") {
-        x /= 18;
+    let scale;
+    // `mu` units scale with scriptstyle/scriptscriptstyle.
```

Comment:
Well, not for all fonts—textstyle in some fonts has a different quad.

For example, under `\tiny`, textstyle is `cmr5`, which has quad:

```
   (QUAD R 1.361133)
```

If the question is referring to the `mu units scale` comment, that comment is trying to explain why the "mu" branch has no `unitOptions`.